### ISSUE TITLE
Delay JavaScript Execution compatibility with Premium Addons for Elementor Pro

### ISSUE THREAD

**Issue by DahmaniAdame** [wp-media/wp-rocket#4416]
**Before submitting an issue please check that you’ve completed the following steps:**
- [x] Made sure you’re on the latest version `3.9.4.1`
- [x] Used the search feature to ensure that the bug hasn’t been reported before

**Describe the bug**
Some Premium Addons for Elementor feature break when Delay JavaScript Execution is used. They won't fire/work properly after user interaction.

For instance: 

**The filter system won't work if delayed.** 

Excluding the following fixes the issue: 

```
/jquery-?[0-9.](.*)(.min|.slim|.slim.min)?.js
/jquery-migrate(.min)?.js
/premium-addons-for-elementor/assets/frontend/min-js/premium-addons.min.js
/premium-addons-for-elementor/assets/frontend/min-js/isotope.min.js
```

**Lottie animations won't fire after user interaction if delayed.**

Excluding the following fixes the issue: 

```
/jquery-?[0-9.](.*)(.min|.slim|.slim.min)?.js
/jquery-migrate(.min)?.js
window.scopes_array
lottie.min.js
```

**To Reproduce**
1. Use Elementor + Premium Addons for Elementor Pro
2. Build gallery with filter and add lottie images
3. See error

**Expected behavior**
Both the filter and Lottie animations should work if delayed.

**Screenshots**
N/A

**Additional context**
Ticket - https://secure.helpscout.net/conversation/1646707348/298186/
Exclusions already adding to our Delay JavaScript Execution compatible help article - https://docs.wp-rocket.me/article/1560-delay-javascript-execution-compatibility-exclusions#premium-addons-for-elementor

**Backlog Grooming (for WP Media dev team use only)**
- [ ] Reproduce the problem
- [ ] Identify the root cause
- [ ] Scope a solution
- [ ] Estimate the effort

**Comment by NataliaDrause** [comment]
Related: https://secure.helpscout.net/conversation/1764897984/320838/

**Comment by piotrbak** [comment]
Will be fixed in the 3.13 backend

**Comment by piotrbak** [comment]
Handled in 3.13 using dynamic lists
